contrib/cirrus/lib.sh

```diff
@@ -55,7 +55,7 @@ PACKER_VER="1.3.5"
 # CSV of cache-image names to build (see $PACKER_BASE/libpod_images.json)
 
 # Base-images rarely change, define them here so they're out of the way.
-export PACKER_BUILDS="${PACKER_BUILDS:-ubuntu-18,fedora-30,fedora-29}"
+export PACKER_BUILDS="${PACKER_BUILDS:-ubuntu-18,xfedora-30,fedora-30,fedora-29}"
```

Comment:
Is this a debugging holdover? I don't see any explanation of this xfedora, nor any subsequent use of it

Reply:
Ohh, how'd that get there.  That's from https://github.com/containers/libpod/pull/3607

Maybe a rebase problem or I need to make this depend on that.  Thanks for noticing.

Reply:
Yep, this is a typo.  I must have forgotten to switch branches.